src/tabs/tabs.js

```diff
@@ -44,6 +50,7 @@ export default class Tabs extends React.Component<TabsPropsT> {
         $orientation: orientation,
         onSelect: () => this.onChange({activeKey: key}),
         children: child.props.title,
+        overrides,
```

Comment:
wonder if we should merge it with `child.props.overrides` with latest takes precedence

Reply:
that's a good point - I have to do that, thanks Nadiia